money-monkey-web/app/lib/api.ts

```
@@ -30,38 +31,50 @@ export async function getCategories(): Promise<Category[]> {
+  year: number
+): Promise<CategorySum[]> {
   return getCategorySums(0, year);
 }
 
-export async function getCategorySums(month: number, year: number): Promise<CategorySum[]> {
-  var userId = auth().userId
+export async function getCategorySums(
+  month: number,
+  year: number
+): Promise<CategorySum[]> {
+  var userId = auth().userId;
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)